src/main/java/com/personal/financeManager/documentProcessor/services/impl/DataEntityServiceImpl.java

```diff
@@ -6,16 +6,22 @@
 import com.personal.financeManager.documentProcessor.models.DataEntityBean;
 import com.personal.financeManager.documentProcessor.models.SuccessResponse;
 import com.personal.financeManager.documentProcessor.services.DataEntityService;
+import com.personal.financeManager.exceptions.PayloadValidationException;
 
 @Service
 public class DataEntityServiceImpl implements DataEntityService{
 
 	@Override
 	public SuccessResponse createDataEntity(DataEntityBean dataEntityBean) {
 		
+		if (dataEntityBean.getDataEntityID() == null || dataEntityBean.getDataEntityID().trim().isEmpty()) {
+            dataEntityBean.setDataEntityID(Long.toString(System.currentTimeMillis()));
+        }
+		
 		JSONObject data = new JSONObject(dataEntityBean.getData());
 		
-		String effectiveFromTimestamp = Long.toString(System.currentTimeMillis());
+		String DataEntityID = Long.toString(System.currentTimeMillis());
```

Comment:
@DevMrunalM Don't understand why you created a new variable **DataEntityID**? As you have already assigned a value using the setter on line number 18, you just wanted to use that.
and second thing, the **variable name should start in lowercase**. i.e. camel case.

Reply:
My mistake ,I failed to remove the variable......will improve to avoid such mistake next time